net/dialer.go

```diff
@@ -41,8 +39,7 @@ func (s *server) dial(peerID libpeer.ID) (pb.ServiceClient, error) {
 	conn, ok := s.conns[peerID]
 	if ok {
 		if conn.GetState() == connectivity.Shutdown {
-			if err := conn.Close(); err != nil && status.Code(err) != codes.Canceled {
-				// log.Errorf("error closing connection: %v", err)
+			if err := conn.Close(); err != nil {
```

Comment:
I tried to test for the situation where this would fail because the code would be `Canceled` and I couldn't make it so. I figured it was dead code and removed it.